File: system-test/java/src/test/java/com/optivem/eshop/systemtest/legacy/mod07/smoke/system/ShopSmokeTest.java
package com.optivem.eshop.systemtest.legacy.mod07.smoke.system;

import com.optivem.eshop.systemtest.legacy.mod07.base.BaseUseCaseDslTest;
import com.optivem.eshop.dsl.channel.ChannelType;
import com.optivem.testing.Channel;
import org.junit.jupiter.api.TestTemplate;

class ShopSmokeTest extends BaseUseCaseDslTest {
    @TestTemplate
    @Channel({ChannelType.UI, ChannelType.API})
    void shouldBeAbleToGoToShop() {
        app.shop().goToShop()
                .execute()
                .shouldSucceed();
    }
}

File: system-test/dotnet/SystemTests/Legacy/Mod08/SmokeTests/System/ShopSmokeTest.cs
using SystemTests.Legacy.Mod08.Base;
using Optivem.Testing;
using Xunit;

namespace SystemTests.Legacy.Mod08.SmokeTests.System;

public class ShopSmokeTest : BaseScenarioDslTest
{
    [Theory]
    [ChannelData(ChannelType.UI, ChannelType.API)]
    public async Task ShouldBeAbleToGoToShop(Channel channel)
    {
        await Scenario(channel)
            .Assume().Shop().ShouldBeRunning();
    }
}
